### DISCUSSION TITLE
Does Prisma has some toSql method

### DISCUSSION THREAD

**Discussion by calebeaires** [prisma/prisma#6646] [Q&A] [answered]
Sometimes it's good to know what exactly the query looks like. Does Prisma offer some type of print SQL version to the object schema? I did not found it in the documentation.

**Comment by 2color** [comment, marked as answer]
Hey @calebeaires,

You can configure Prisma to log the SQL queries when you [instantiate PrismaClient](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#options), for example:
```
const prisma = new PrismaClient({ log: ['query'] })
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
